### ISSUE TITLE
Move affixes "inside" input

### ISSUE THREAD

**Issue by zepfietje** [laravel-filament/filament#14873]
Make sure clicking prefix/suffix icon focuses input.

**Comment by danharrin** [comment]
@zepfietje please can you explain what you mean by "inside"? I understand that clicking should focus the input though

**Comment by zepfietje** [comment]
By "inside" I mean absolutely positioning the affixes to ensure that clicking them focuses the input. There's probably some tricky parts to different approaches though.

**Comment by sinnbeck** [comment]
A small trick would be to simply wrap it all in a `<label>` tag. That will make it all clickable to focus.

https://jsfiddle.net/t5xbpuLm/

**Comment by danharrin** [comment]
I don't think we can do that without affecting accessibility @sinnbeck. To be honest I think the only way this will be able to happen without introducing complex styling challenges is by adding click handlers to the input-wrapper to focus the input inside somehow.
